### PR TITLE
LPS-141499 Display default name of Objects and its Fields, Relationships, Layouts if there isn't translation for the selected Portal language

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-141499

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Review by marcelabc — CHANGES_REQUESTED** [review]
I finished my review with one question related to the code implementation.

File: modules/apps/object/object-api/src/main/java/com/liferay/object/util/LocalizedMapUtil.java [hunk 1]
```diff
@@ -25,8 +26,43 @@
  */
 public class LocalizedMapUtil {
 
+	public static Map<String, String> getLanguageIdMap(
```
[the remark this answers is not in the document]
Comment (p-albuquerque) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/util/LocalizedMapUtil.java]: Done.

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:stop

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**